proto-lens-runtime/package.yaml

```diff
@@ -1,5 +1,5 @@
-name: proto-lens-runtime
-version: '0.7.0.3'
+gname: proto-lens-runtime
```

Comment:
Is this a typo, `gname`?

Reply:
Ouch.  I'll pretend I was checking if you were awake :)

Reply:
lol. I'll merge once the CI passes.